src/flow.h

```diff
@@ -302,6 +302,40 @@ typedef struct FlowCnf_
     SC_ATOMIC_DECLARE(uint64_t, memcap);
 } FlowConfig;
 
+#if defined(ENABLE_ETM)
+
+#define FLOW_SPLT_MAX_COUNT       32
```

Comment:
general comment for the defines and structs: add some documentation of that the fields and values are